packages/dds/tree/src/test/codec/ajvValidator.ts

```diff
@@ -22,6 +18,7 @@ const Ajv =
 	(ajvModuleOrClass as any);
 
 import type { JsonValidator } from "../../codec/index.js";
+import { mockSerializer } from "../mockSerializer.js";
```

Comment:
other tests may need to use this in the future, so separate it into its own file.